### PR TITLE
ClaimGuid -> StatementGuid

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
As per https://github.com/wmde/WikibaseDataModel/issues/412

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Will update rell notes in follow up change as usual.

File: src/Statement/StatementGuidParsingException.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Wikibase\DataModel\Statement;
+
+/**
+ * @since 0.5
```
Comment (Benestar) [inline src/Statement/StatementGuidParsingException.php]: Maybe add a note that this was renamed in 3.0?

**Comment by Benestar** [comment]
+1 not sure if the inline `@since` tags should mention the rename though.

**Comment by JeroenDeDauw** [comment]
Good catch, will amend

**Comment by mariushoch** [comment]
We really need to make sure we don't forget about replacing all the new class aliases in Wikibase before this hits production. (cc @filbertkm )

**Pull request merged**